corehq/apps/users/models.py

```diff
@@ -1521,6 +1521,8 @@ def save(self, fire_signals=True, **params):
             if self._rev and not self.to_be_deleted():
                 django_user = self.sync_to_django_user()
                 django_user.save()
+                if params.get('unretired_by'):
+                    self.log_user_create(params['unretired_by'], params.get('unretired_via'))
```

Comment:
seems odd to use `log_user_create` when retiring a user

Reply:
this is "unretire" @snopoke . When a user is re-activated, we re-create the django user.

Reply:
fiar enough, either way I don't think an 'unretiring' is equivalent to create. It should be logged as a change.

Reply:
Agree its a change for the CouchUser but we are tracking everything through `log_model_change` as changes to/by the django user which i think can get confusing.